src/components/06_wrappers/Default/Default.js

```diff
@@ -117,12 +117,14 @@ Default.propTypes = {
   error: string,
   menuLinks: objectOf(
```

Comment:
Could we do this on the top level too?

Reply:
Wait, isn't https://github.com/jsdrupal/drupal-admin-ui/pull/57/files#diff-8b5084ad240e61c05a90cbae6a377e86R101 doing that?